keyboards/ergodox_stm32/matrix.c

```diff
@@ -0,0 +1,196 @@
+#include <stdint.h>
```

Comment:
I don't know how much of this code is based on the Ergodox EZ code, sinc I haven't taken a close look side by side here ....  

But there have been some significant changes to the ergodox ez's custom matrix to allow for better debouncing.  You may want to review those changes and implement them here, as well.   Though, I would recommend the default debounce type.